advanced/sr_gui_bootloader/src/sr_gui_bootloader/bootloader.py

```diff
@@ -306,7 +336,6 @@ def get_widget(self):
 
 
 if __name__ == "__main__":
-    rospy.init_node("bootloader")
```

Comment:
Why is this removed?

Reply:
Reverted (removed this accidentally after some tests)